### DISCUSSION TITLE
What's the best way to export filtered data from nzTable

### DISCUSSION THREAD

**Discussion by radiumrasheed** [NG-ZORRO/ng-zorro-antd#6438] [Q&A]
I have a table with server-side filtering & pagination, however, I want to export the data on the client to CSV or excel. 

I know we have 3rd party library for this but which works best with nzTable without much configuration?

**Comment by dev-npm** [comment]
@radiumrasheed   did you ever figure this out, I am running into same situation.    what 3rs party library did you end up using

**Reply by radiumrasheed** [reply]
hi, I can't remember what project I was working on to check this
If I do, I'll let you know
